src/main/java/com/metamx/emitter/core/factory/EmitterFactory.java

```diff
@@ -0,0 +1,11 @@
+package com.metamx.emitter.core.factory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.metamx.common.lifecycle.Lifecycle;
+import com.metamx.emitter.core.Emitter;
+import com.metamx.http.client.HttpClient;
+
+public interface EmitterFactory
```

Comment:
See https://github.com/druid-io/druid/blob/druid-0.9.2/server/src/main/java/io/druid/client/cache/CacheProvider.java for suggested jackson friendly annotations that can be used.